<launch>

	<node pkg="turtlesim" type="turtlesim_node" name="turtlesim" output="screen"/>

	<node pkg="amazing_turtles" type="action_race" name="controller_2" output="screen">
	</node>



</launch>
